citations/citations-util/util/src/bundle/citations.properties

```diff
@@ -227,8 +227,8 @@ no.results			= No results found to match your query
 title.results		= Search Results
 add.results			= Add
 add_alt.results		= Add to citation list
-remove.results		= Remove
```

Comment:
Can you revert these changes? You're updating the default locale.

Reply:
Indeed, this is a double key value.